unittests/test_utility.py

```diff
@@ -976,3 +976,22 @@ def test_reversed(self):
 
         s = util.OrderedSet(l)
         assert list(reversed(s)) == list(reversed(l))
+
+    def test_concat_files(self):
+        tmpdir = tempfile.mkdtemp(dir='unittests')
+        _, file1 = tempfile.mkstemp(dir=tmpdir)
+        _, file2  = tempfile.mkstemp(dir=tmpdir)
+        _, concat_file = tempfile.mkstemp(dir=tmpdir)
```

Comment:
Since you are already in a unique temporary directory, you need not create temporary files for these. Just normal names, such as `in1.txt`, `in2.txt` etc. should be fine.